common/djangoapps/util/password_policy_validators.py

```diff
@@ -320,3 +320,11 @@ def _validate_password_dictionary(value):
             if edit_distance <= password_max_edit_distance:
                 raise ValidationError(_("Password is too similar to a dictionary word."),
                                       code="dictionary_word")
+
+
+def normalize_password(password):
```

Comment:
Nit: Harder to see until you rebase, but I think it would be easier to read if this function were at the top of the file with the other util functions.